src/Stratis.Bitcoin.IntegrationTests/Wallet/SendingStakedCoinsBeforeMaturity_Steps.cs

```diff
@@ -74,7 +74,8 @@ private IActionResult SendTransaction(IActionResult transactionResult)
             if (walletTransactionModel == null)
                 return null;
 
-            return this.proofOfStakeSteps.PremineNodeWithCoins.FullNode.NodeController<WalletController>().SendTransaction(new SendTransactionRequest(walletTransactionModel.Hex));
+            return this.proofOfStakeSteps.PremineNodeWithCoins.FullNode.NodeController<WalletController>()
+                .SendTransaction(new SendTransactionRequest(walletTransactionModel.Hex)).GetAwaiter().GetResult();
```

Comment:
Same here, please check this file and remove the `GetAwaiters`

Reply:
I think we need them unless we make a refactor of the test to make the whole chain async

Reply:
Done